watson-language/watson-lang.lua

```diff
@@ -0,0 +1,95 @@
+--[[
+
+# [Salesfors API](https://console.bluemix.net/docs/services/language-translator/getting-started.html) connector.
+
+### Authorization
+
+Authorization use [Public]().
```

Comment:
you don't need empty link, just use plain text "There is no authorization required".